bootstrap-scripts/bootstrap.sh

```diff
@@ -285,6 +308,10 @@ if [[ "$INSTALL_SDK_HEADERS" == '1' ]]; then
   fi
 fi
 
+if [[ "$BREW_INSTALL_LIBFFI" == "1" ]]; then
+  echo "brew 'libffi'" >> Brewfile
```

Comment:
Not the greatest for immutable `Brewfile`... maybe I'll try some ruby detection inside default Brewfile?